### PR TITLE
fix(puma): RHICOMPL-3223 make the puma control socket path static

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: config/puma.rb [hunk 1]
```diff
@@ -44,6 +44,6 @@
 plugin :tmp_restart
 
 # Metrics
-activate_control_app
+activate_control_app('unix://./tmp/sockets/puma.sock')
```
Comment (vkrizan) [inline config/puma.rb:47]: Would this work for forked workers?

Comment (skateman) [reply config/puma.rb:47]: Yes, puma has this solved for forked workers, you only control the parent process and not the children.

Comment (vkrizan) [reply config/puma.rb:47]: `./` bounds it to CWD. Shouldn't it be specific apps relative location?

Comment (skateman) [reply config/puma.rb]: CWD here is the specific app's relative location, Rails.root is not available in this context. Do you have any other suggestions how to put it into the `./tmp` folder?

Comment (vkrizan) [reply config/puma.rb]: Maybe from the file location?

Comment (skateman) [reply config/puma.rb]: Seems like an overkill, but okay

**Pull request merged**